tooling/eslint/nextjs.js

```diff
@@ -3,6 +3,7 @@ const config = {
   extends: ["plugin:@next/next/recommended"],
   rules: {
     "@next/next/no-html-link-for-pages": "off",
+    "@typescript-eslint/require-await": "off",
```

Comment:
this rule is kinda annoying for nextjs app router as some stuff jsut has to be async no matter if they use await or not...